crates/apub/src/objects/post.rs

```diff
@@ -287,16 +267,6 @@ impl Object for ApubPost {
       context.reset_request_count(),
     );
 
-    // write mod log entry for lock
-    if Page::is_locked_changed(&old_post, &page.comments_enabled) {
-      let form = ModLockPostForm {
-        mod_person_id: creator.id,
-        post_id: post.id,
-        locked: Some(post.locked),
-      };
-      ModLockPost::create(&mut context.pool(), &form).await?;
-    }
-
```

Comment:
This might need to be kept.

Reply:
good catch.

Reply:
Its not necessary here anymore as posts cant be locked this way. Instead it uses the [lock page activity](https://github.com/LemmyNet/lemmy/pull/4717/files#diff-aa26b41304fc5ced54331040b0ac0a9f7038bf53f30736acaa86da6f86185e26). But I just noticed that it didnt write the mod log at all, so Ive added that now.